src/Auth.php

```diff
@@ -271,19 +280,26 @@ public function connection_ldap($ldap_method, $login, $password)
                 ]);
             } catch (\Throwable $e) {
                 ErrorHandler::getInstance()->handleException($e, true);
+                $info = false;
+            }
+
+            $ldap_errno = ldap_errno($this->ldap_connection);
+            if ($info === false && $ldap_errno > 0 && $ldap_errno !== 32) {
+                // 32 = LDAP_NO_SUCH_OBJECT => This should not be considered as a connection error, as it just means that user was not found.
```

Comment:
Indeed, this is not a connection error, but we know at this point this is already NOK.
I would have expected the code to stop properly here if user is not found; am I wrong?